docs-web/src/main/webapp/src/locale/en.json

```diff
@@ -286,6 +286,7 @@
       "enabled": "Enable LDAP authentication",
       "host": "LDAP hostname",
       "port": "LDAP port (389 by default)",
+      "usessl": "Aktiviere SSL (ldaps)",
```

Comment:
it's the english translation file => "Enable SSL (ldaps)"

Reply:
fixed. overseen it. thanks!